helm-charts/splunk-otel-collector/values.schema.json

```diff
@@ -128,6 +782,208 @@
+              "properties": {
+                "cpu": {
+                  "type": [
+                    "string",
+                    "integer"
+                  ]
+                },
+                "memory": {
+                  "type": "string"
+                }
+              }
+            },
+            "requests": {
+              "type": "object",
+              "additionalProperties": false,
+              "properties": {
+                "cpu": {
+                  "type": [
+                    "string",
+                    "integer"
+                  ]
+                },
+                "memory": {
+                  "type": "string"
+                }
+              }
+            }
+          }
+        },
+        "nodeSelector": {
+          "type": "object"
+        },
+        "tolerations": {
+          "type": "array"
+        },
+        "affinity": {
+          "type": "object"
+        },
+        "securityContext": {
+          "type": "object"
+        },
+        "terminationGracePeriodSeconds": {
+          "type": "integer"
+        },
+        "priorityClassName": {
+          "type": "string"
+        },
+        "annotations": {
+          "type": "object"
+        },
+        "podAnnotations": {
+          "type": "object"
+        },
+        "podLabels": {
+          "type": "object"
+        },
+        "extraEnvs": {
+          "type": "array",
+          "items": {
+            "type": "object"
+          }
+        },
+        "extraVolumes": {
+          "type": "array",
+          "items": {
+            "type": "object"
+          }
+        },
+        "extraVolumeMounts": {
+          "type": "array",
+          "items": {
+            "type": "object"
+          }
+        },
+        "config": {
+          "type": "object"
+        }
+      }
+    },
+    "service": {
+      "description": "Splunk OpenTelemetry collector service configuration.",
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "type": {
+          "type": "string",
+          "enum": [
+            "ClusterIP",
+            "NodePort",
+            "LoadBalancer",
+            "ExternalName"
+          ]
+        },
+        "annotations": {
+          "type": "object"
+        }
+      }
+    },
+    "splunkRealm": {
+      "description": "Splunk Observability Realm",
+      "type": "string"
+    },
+    "splunkAccessToken": {
+      "description": "Splunk Observability Access Token",
+      "type": "string"
+    },
+    "ingestUrl": {
+      "type": "string"
+    },
+    "apiUrl": {
+      "type": "string"
+    },
+    "metricsEnabled": {
+      "description": "Send Metrics to Splunk Observability",
+      "type": "boolean"
+    },
+    "tracesEnabled": {
+      "description": "Send Traces to Splunk Observability",
+      "type": "boolean"
+    },
+    "logsEnabled": {
+      "description": "Send Logs to Splunk Observability",
+      "type": "boolean"
```

Comment:
Please add `deprecated: true` attribute and `[DEPRECATED] ` description prefix to all of these parameters